opta/commands/kubectl.py

```diff
@@ -16,7 +16,14 @@
 @click.option(
     "-e", "--env", default=None, help="The env to use when loading the config file"
 )
-def configure_kubectl(config: str, env: Optional[str]) -> None:
+@click.option(
+    "-f",
+    "--force",
+    is_flag=True,
+    default=False,
+    help="Purge the existing cached kube config",
+)
```

Comment:
what about making this the default behavior so this option is not needed?

Reply:
good point